manifests/plugin.pp

```diff
@@ -1,9 +1,12 @@
-define minecraft::plugin($plugin_name = $title, $source) {
+define minecraft::plugin(
+  $source,
+  $plugin_name  = $title
+) {
 
   if $plugin_name =~ /^.*\.jar$/ {
     fail("minecraft plugin name ${plugin_name} must not end in '.jar'")
```

Comment:
Ya, we're already checking if it ends in `.jar`, if it doesn't just tack it on... But that should probably be part of a different PR.